app/views/todos/index.html.erb

```diff
@@ -2,8 +2,8 @@
 
 <div id="user_space">
   <p>
-  Login with email: <%= current_user.email %>
-  <%= current_user.id %>
+  <%#= current_user.email %>
```

Comment:
Remove commented lines, if you need those in a future you can check removed code in the git history